pytorch/src/main/java/org/bytedeco/pytorch/presets/torch.java

```diff
@@ -73,30 +79,51 @@
                 "torch/csrc/jit/serialization/storage_context.h",
 
                 "datasets.h",
-                "pytorch_adapters.h"
+                "pytorch_adapters.h",
+
+		        // Fix link error on Windows:
+		        "gloo/common/logging.cc"
+
             },
             exclude = {"openblas_config.h", "cblas.h", "lapacke_config.h", "lapacke_mangling.h", "lapack.h", "lapacke.h", "lapacke_utils.h"},
-            link = {"c10", "torch_cpu", "torch"},
-            preload = {"gomp@.1", "iomp5", "omp", "tbb@.2", "asmjit", "fbgemm"}
+            link = { "c10", "torch", "torch_cpu" }
         ),
         @Platform(
             value = {"linux", "macosx", "windows"},
-            link = { "c10", "c10_cuda", "torch_cpu", "torch_cuda", "torch" },
-            preload = {"gomp@.1", "iomp5", "omp", "tbb@.2", "asmjit", "fbgemm", "cupti@.12"},
```

Comment:
Please put back the preloads here and above

Reply:
preloads were moved to platform-specific sections, because the lib to preloads differ for each platform, in particular the openmp library. It's cleaner than trying to load everything for each platform. Is there a problem with that ? 
Do you spot a library that should be preloaded and is not anymore ?
gomp is not in the linux jar. 
iomp5 is for macos only
omp is not used by any platform anymore
tbb is not used
asmjit and fbgemm exist for windows only
cupti is now handled by inherited cuda preset

Reply:
Add the missing one like gomp at least yes

Reply:
gomp added, but I don't think it has any effect since there is no jar containing libgomp.

Reply:
If it doesn't get bundled, that's a bug

Reply:
It does, sorry. It didn't with my local build for some reason but it's bundled by the github runner.